rita/src/main.rs

```diff
@@ -21,25 +18,25 @@ use babel_monitor::Babel;
 extern crate traffic_watcher;
 
 extern crate debt_keeper;
-use debt_keeper::{Key, DebtKeeper};
+use debt_keeper::{DebtKeeper, DebtAction, DebtAdjustment, Identity};
 
 extern crate payment_controller;
-use payment_controller::{Payment, PaymentController};
-
-extern crate rita_types;
-use rita_types::DebtAction;
+use payment_controller::{PaymentTx, PaymentController};
 
 extern crate docopt;
 use docopt::Docopt;
 
 extern crate ip_network;
 extern crate simple_logger;
 
+extern crate tunnel_manager;
+use tunnel_manager::TunnelManager;
+
 extern crate num256;
 use num256::Int256;
 
-extern crate rouille;
-use rouille::{Request, Response};
+#[macro_use] extern crate rouille;
```

Comment:
People usually write `#[macro_use]` above `extern crate`, does this build normally?

Reply:
As far as I can tell, yes